frontend/src/__tests__/cypress/cypress/pages/permissions.ts

```diff
@@ -29,6 +29,18 @@ class PermissionsTab {
   getGroupTable() {
     return new PermissionTable(() => cy.findByTestId('role-binding-table Group'));
   }
+
+  findConfirmModal() {
+    return cy.findByTestId('role-binding-permissions-change-modal');
+  }
+
+  findModalCancelButton() {
+    return cy.findByTestId('cancel-button');
+  }
+
+  findModalConfirmButton() {
+    return cy.findByTestId('confirm-button');
+  }
```

Comment:
Other option is to move these to their own class and call it something like `PermissionsModal` and extend the base Modal. Add the confirm button and some sort of is visible.